src/components/SignUp/SignUp.scss

```diff
@@ -0,0 +1,39 @@
+@import '../../style/constants';
+
+label {
+  margin-top: 1rem !important;
```

Comment:
Is the use of this `!important` really necessary? Can you avoid it using a class or more specificity?